core/src/main/scala/com/salesforce/op/ModelInsights.scala

```diff
@@ -647,6 +670,18 @@ case object ModelInsights {
     }
   }
 
+  private[op] def checkLRStandardization(model: Option[Model[_]]): Option[Boolean] = {
+    val stage = model.flatMap {
```

Comment:
```scala
for {
   m <- model
   stage <- m.getSparkMlStage()
} yield stage match {
    case s: LogisticRegressionModel | LinearRegressionModel => s.getStandardization
    case _ => false
}
```

Reply:
`m.getSparkMlStage()` the compiler is not happy about this

Reply:
the compiler is not happy with `m.getSparkMlStage()`

Reply:
either change the type of the function argument to `model: Option[Model[_] with SparkWrapperParams])` or add collect:
```scala
for {
   m <- model.collect { case v: SparkWrapperParams => v }
   stage <- m.getSparkMlStage()
} yield stage match {
    case s: LogisticRegressionModel | LinearRegressionModel => s.getStandardization
    case _ => false
}
```